freeclimb-cs-sdk/api/call/Call.cs

```diff
@@ -345,6 +355,7 @@ public static bool Equal(Call a, Call b)
                    String.Equals(a.getTo, b.getTo, StringComparison.Ordinal) &&
                    String.Equals(a.getFrom, b.getFrom, StringComparison.Ordinal) &&
                    String.Equals(a.getPhoneNumberId, b.getPhoneNumberId, StringComparison.Ordinal) &&
+                   (a.getActive == b.getActive) &&
```

Comment:
I don't think this is needed. Active is not a stored call property but is set to true or false depending on the status (which we already compare here).